minutes-of-meetings/2022-12-13-Tue - Minutes.md

```diff
@@ -0,0 +1,46 @@
+# Meeting on 2022-12-13 (Tuesday)
+
+## Logistics
+
+- Where: Online via [Meetup](https://www.meetup.com/wardley-mapping-foundation/events/289644918/)
+- When: 15:00 to 15:30 GMT
+- Attending:  [Steve Purkis](https://twitter.com/spurkis), [Ann Bergin](https://twitter.com/agvbergin), [Catherine Walker](https://twitter.com/transageo)
+
+## Follow-ups and direction
+
+- [ ] @spurkis to do calendar invite for collaborative editing on Friday, 3pm.
+- [ ] @spurkis to put together shortlist of trustees to approach in order for Friday collaboration session.
+- [ ] @agvbergin to tidy-up her comments on the doc for trustees before the Friday collaboration session.
+- [ ] @agvbergin to add objects from purpose to the articles of association doc before the Friday collaboration session.
+
+## Discussions
+
+https://github.com/Wardley-Mapping-Foundation/wmf-operating-model/issues/55
+
+Needs thank you note writing for those who contributed, still at same point as last week.
+
+https://github.com/Wardley-Mapping-Foundation/wmf-operating-model/issues/56
+
+Agreed to close, with minutes more useful than a full transcript. Noted that transcription useful as a minuting tool, not published directly.
+
+https://github.com/Wardley-Mapping-Foundation/wmf-operating-model/issues/51
+
+Agreed to close, summary on ticket sufficient. We note that if we received donations outside the UK, we'd likely have to deal with the bureaucracy of this ourselves.
+
+https://github.com/Wardley-Mapping-Foundation/wmf-operating-model/issues/30
+
+- How will we know when the doc is at a satisfactory level, and who decides? Need to finish items raised via comments on the document.
+- Catherine will look at doc after call.
+- Arrange collaborative editing after 3pm on Friday, in General Discord channel, @spurkis to do calendar invite.
+
+https://github.com/orgs/Wardley-Mapping-Foundation/projects/6/views/1#
```

Comment:
do we need this link to the github project? Or did you have in mind adding a link to issue 63?

Reply:
Apologies, I was running the project board and trying to write minutes simultaneously ... this should be issue 73.